app/Http/Controllers/API/SendController.php

```diff
@@ -178,22 +178,22 @@ protected function executeSend(APIControllerHelper $helper, Request $request, Pa
                     $allow_unconfirmed = isset($request_attributes['unconfirmed']) ? $request_attributes['unconfirmed'] : false;
                     // Log::debug("\$allow_unconfirmed=".json_encode($allow_unconfirmed, 192));
 
-                    // validate that the funds are available, bypass this is custom_inputs used
+                    // validate that the funds are available, bypass this if custom_inputs used
                     if(!$custom_inputs){
-						if ($allow_unconfirmed) {
-							$has_enough_funds = AccountHandler::accountHasSufficientFunds($account, $float_quantity, $asset, $float_fee, $dust_size);
-						} else {
-							$has_enough_funds = AccountHandler::accountHasSufficientConfirmedFunds($account, $float_quantity, $asset, $float_fee, $dust_size);
-						}
-						if (!$has_enough_funds) {
-							EventLog::logError('error.send.insufficient', ['address_id' => $payment_address['id'], 'account' => $account_name, 'quantity' => $float_quantity, 'asset' => $asset]);
-							return new JsonResponse(['message' => "This account does not have sufficient".($allow_unconfirmed ? '' : ' confirmed')." funds available."], 400);
-						}
-					}
+                        if ($allow_unconfirmed) {
+                            $has_enough_funds = AccountHandler::accountHasSufficientFunds($account, $float_quantity, $asset, $float_fee, $dust_size);
+                        } else {
+                            $has_enough_funds = AccountHandler::accountHasSufficientConfirmedFunds($account, $float_quantity, $asset, $float_fee, $dust_size);
+                        }
+                        if (!$has_enough_funds) {
+                            EventLog::logError('error.send.insufficient', ['address_id' => $payment_address['id'], 'account' => $account_name, 'quantity' => $float_quantity, 'asset' => $asset]);
+                            return new JsonResponse(['message' => "This account does not have sufficient".($allow_unconfirmed ? '' : ' confirmed')." funds available."], 400);
+                        }
+                    }
 
 
                     // send the funds
-                    EventLog::log('send.begin', ['request_id' => $request_id, 'address_id' => $payment_address['id'], 'account' => $account_name, 'quantity' => $float_quantity, 'asset' => $asset, 'destination' => ($is_multisend ? $destinations : $destination), 'custom_inputs' => json_encode($custom_inputs)]);
+                    EventLog::log('send.begin', ['request_id' => $request_id, 'address_id' => $payment_address['id'], 'account' => $account_name, 'quantity' => $float_quantity, 'asset' => $asset, 'destination' => ($is_multisend ? $destinations : $destination), 'custom_inputs' => $custom_inputs]);
```

Comment:
The logger can handle arrays, so you don't need to `json_encode` any data sent to `EventLog::log`.